modules/tinymce/src/core/main/ts/dom/ElementUtils.ts

```diff
@@ -112,8 +122,12 @@ const ElementUtils = (dom: DOMUtils): ElementUtils => {
     return !Bookmarks.isBookmarkNode(node1) && !Bookmarks.isBookmarkNode(node2);
   };
 
+  const attributeIsInternal = (attributeName: string): boolean =>
```

Comment:
Just to follow normal naming standards/conventions, maybe call this `isAttributeInternal` as it's common to start functions like this with `is`.